store/demo/dev/10130__issue.sql

```diff
@@ -274,4 +274,32 @@ VALUES
         102
     );
 
-ALTER SEQUENCE issue_id_seq RESTART WITH 13010;
+-- Issue for task dependency
+INSERT INTO
+    issue (
+        id,
+        creator_id,
+        updater_id,
+        project_id,
+        pipeline_id,
+        name,
+        status,
+        type,
+        description,
+        assignee_id
+    )
+VALUES
+    (
+        13010,
+        1,
+        1,
+        3001,
+        9010,
+        'Hello task dependency!',
+        'OPEN',
+        'bb.issue.general',
```

Comment:
This should be bb.issue.database.schema.update ?

Reply:
Done.